org.civicrm.reqangular/test/directives/horizontal-scroll-shadows.directive.spec.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/angular',
+  'common/lodash',
+  'common/angularMocks',
+  'common/directives/horizontal-scroll-shadows.directive'
+], function (angular, _) {
+  'use strict';
+
+  describe('horizontalScrollShadows directive', function () {
+    var $compile, $document, $rootScope, $window, element, scope;
+
+    beforeEach(module('common.directives'));
+    beforeEach(inject(function (_$compile_, _$document_, _$rootScope_, _$window_) {
+      $compile = _$compile_;
+      $document = _$document_;
+      $rootScope = _$rootScope_;
+      $window = _$window_;
+    }));
+
+    describe('when "horizontal-scroll-shadows" attribute is specified', function () {
+      var content, wrapper;
+
+      beforeEach(function () {
+        scope = $rootScope.$new();
+        element = angular.element('<div><div horizontal-scroll-shadows>Content</div></div>');
+
+        $compile(element)(scope).controller('horizontalScrollShadows');
+
+        $document.find('body').append(element);
+
+        content = $document.find('.horizontal-scroll-shadows-content');
+        wrapper = $document.find('.horizontal-scroll-shadows-wrapper');
+
+        // Overflow is needed to be set to "auto", otherwise scroll won't work
+        wrapper.css({ 'overflow-x': 'auto' });
+      });
+
+      describe('when not scrollable', function () {
```

Comment:
more descriptive please. When the content is not scrollable, otherwise it seems that you're talking about the directive

Reply:
Done